_includes/release-notes/51known-issues.md

```diff
@@ -0,0 +1,6 @@
+
+This section describes issues that users of {{site.productname}} 5.1 may encounter, as well as possible workarounds for these issues.
```

Comment:
...is this meant to have more info? or is that coming later?

Reply:
Don't know. It can be removed if no-one wants it.

Reply:
One thing I just thought of, was that when opening UI elements (eg dialogs, context menus/toolbars, etc...) on iOS 12 that they may appear below the keyboard. The reason for this, is the new silver theme relies on the `VisualViewport` API, which is an experimental feature in iOS 12. So if users are using iOS 12 then they'll either need to upgrade, or to turn on the experimental feature by going to Settings -> Safari -> Advanced -> Experimental Features -> Visual Viewport API.

Reply:
Added the issue to know issues